src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +794,86 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        deployment_configs = self.list_deployment_configs()
+
+        data = {"Config Name": [], "Instance Type": [], "Selected": [], "Accelerated": []}
+
+        for i in range(0, len(deployment_configs)):
+            deployment_config = deployment_configs[i]
+            benchmark_metrics = deployment_config.get("benchmark_metrics")
+
+            data["Config Name"].append(deployment_config.get("config_name"))
+            data["Instance Type"].append(
+                deployment_config.get("deployment_config").get("instance_type")
+            )
+
+            # TODO: We should not hard code these
+            if i == 0:
+                data["Selected"].append("Yes")
+            else:
+                data["Selected"].append(" - ")
+
+            # TODO: We should not hard code these
+            if i % 2 == 0:
```

Comment:
is this deterministic? so even rows will always be accelerated instance types, odd rows will always be cpu instance types?

also just curious, what value does this column provide? Shouldn't InstanceType be enough?